authors/apps/authentication/templates/email.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Authors Haven account verification</title>
+    <meta name="description" content="Our first page">
+    <meta name="keywords" content="html tutorial template">
+</head>
+
+
+<body>
+    <div id="div-page" style="border: 2px solid #86C232;border-radius: 25px;width: 30em;padding: 25px;margin: 5em;text-align: justify; border-width: thick">
```

Comment:
Maybe look into having all CSS styles in a class or ID rather than inline in the HTML

Reply:
This will break the email template since it has to be rendered in the views. It will have to remain like that at the moment.